### PR TITLE
Pt/callout

### PR DESCRIPTION
**Pull request by paaaatrick**
## Trello Card(s)
https://trello.com/c/b6Q7bUqm/26-section-callout

## What was updated?
Callout template and styles added.

## Screenshots of updates (visual changes)

Desktop
Screenshot 2019-11-27 16 48 15

Mobile
Screenshot 2019-11-27 16 49 11

### CODE REVIEW

File: web/themes/fire/templates/components/callout/paragraph--callout.html.twig [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        {% include directory ~ "/assets/media/svgs/icon--logo.svg" %}
+      </div>
+      {% if copy %}
+        <div class="callout__text">{{ copy }}</div>
+      {% endif %}
+      {% if link %}
+        <a class="callout__link" href="{{link['#url']}}" >{{ link['#title'] }}</a>
+      {% endif %}
+      <div class="callout__background">
+        {# <div class="lozad callout__background-image"></div> #}
```
Comment (ryanschmidt) [inline web/themes/fire/templates/components/callout/paragraph--callout.html.twig]: what dis?

Comment (paaaatrick) [reply web/themes/fire/templates/components/callout/paragraph--callout.html.twig]: :( oops

File: web/themes/fire/main.scss [hunk 2]
```diff
@@ -10,13 +10,14 @@
...
 @import '~bootstrap/scss/bootstrap-reboot';
 @import '~bootstrap/scss/bootstrap-grid';
-@import '~bootstrap/scss/utilities/display';
 @import '~bootstrap/scss/pagination';
 @import '~bootstrap/scss/alert';
 @import '~bootstrap/scss/forms';
 @import '~bootstrap/scss/buttons';
 @import '~bootstrap/scss/nav';
 @import '~bootstrap/scss/dropdown';
+@import '~bootstrap/scss/utilities/display';
```
Comment (1forh) [inline web/themes/fire/main.scss]: didn't know you could do this. seems like a good idea.

File: web/themes/fire/templates/components/callout/_callout.scss [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      height: 31px;
+      width: 31px;
+
+      path {
+        fill: theme-color('primary');
+      }
+    }
+  }
+
+  &__background {
```
Comment (1forh) [inline web/themes/fire/templates/components/callout/_callout.scss]: css order

File: web/themes/fire/main.scss (hunk 2 above)
Comment (theroadeldorado) [reply web/themes/fire/main.scss]: I think it can only be moved down.

**Pull request merged**